src/main/plugin/iso19139.ca.HNAP/layout/config-editor.xml

```diff
@@ -310,6 +310,8 @@
         <section name="gmd:MD_Metadata">
           <field xpath="/gmd:MD_Metadata/gmd:fileIdentifier"/>
           <field xpath="/gmd:MD_Metadata/gmd:dateStamp"/>
+          <field xpath="/gmd:MD_Metadata/gmd:language" or="language" in="/gmd:MD_Metadata"/>
```

Comment:
`or` and `in` attributes require also the element to be defined in the `flatModeExceptions` list, so the editor adds the `+` button if the element if doesn't exist in the metadata. 

But in this case as in the default view is a readonly element I don't think it's required.

Reply:
Fixed - thank you for the review.